new-frontiers-14/frontier-station-14
# Fireaxe Removes Plating (Delta-V#1700), Maintenance Jack

## About the PR



Cherry-picks [Delta-V#1700](https://github.com/DeltaV-Station/Delta-v/pull/1700) (Thank you, MilonPL).

Adds a maintenance jack as a purchasable tool from the EngiVend machine that uses the new Axing action.  It's a huge item that's carriable on your back or suit storage (though it currently lacks sprites for on-body states).

Adds yellow crowbars to the POI engivend inventory.

## Why / Balance


Convenience, nice to not have to use RCD ammo to remove ship plating.

## How to test


1. Visit the EngiVend, purchase a maintenance jack.
2. Spawn a crowbar.
3. On Frontier or the bus, remove the floor tiling with the crowbar.  Try to remove the plating with the maintenance jack.  You should get a do-after, but nothing should happen.
4. Warp to a POI.
5. Remove the floor tiling with a crowbar, remove the plating with the maintenance jack.  It should remove it, and you should see the bare lattice exposed underneath.

## Media


## Requirements


## Breaking changes


None.

**Changelog**



:cl:
- add: The fireaxe can now remove hull plating.
- add: A new tool, the maintenance jack, is available from the EngiVend. It can remove hull plating.
